### PR TITLE
[KEYCLOAK-10705] - Return full resource representation when querying policies by id

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: services/src/main/java/org/keycloak/authorization/admin/PolicyService.java [hunk 1]
```diff
@@ -176,6 +176,7 @@ public Response findAll(@QueryParam("policyId") String id,
                             @QueryParam("scope") String scope,
                             @QueryParam("permission") Boolean permission,
                             @QueryParam("owner") String owner,
+                            @QueryParam("fields") String fields,
```
Comment (mhajas) [inline services/src/main/java/org/keycloak/authorization/admin/PolicyService.java]: @pedroigor Can this be changed to boolean? Maybe I am missing something but this string is always used only in: `fields != null && fields.equals("*")`.

**Review by mhajas — COMMENTED** [review]
LGTM, I added just one question to the code

File: services/src/main/java/org/keycloak/authorization/admin/PolicyService.java (hunk 1 above)
Comment (pedroigor) [reply services/src/main/java/org/keycloak/authorization/admin/PolicyService.java]: For now yes, but we may in the future use this parameter to let clients choose the fields they want back in the response.

**Review by sguilhen — APPROVED** [review]
Looks good to me

**Review by mhajas — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**